server.js

```diff
@@ -4,6 +4,8 @@ var formidable = require('formidable'),
     port = process.env.PORT || 5000;
     stringify = require('json-stringify-safe');
 
+var DIRECT_UPLOAD_LIMIT = 32;

```

Comment:
It's bytes; this corresponds to the test: https://github.com/apache/cordova-plugin-file-transfer/pull/117/files#diff-2a8a5fef3397df87ab538f028a5c6b50R955 and the contents: https://github.com/apache/cordova-plugin-file-transfer/blob/9e93bad83c9e980cb593dc057fe8af40f35652f1/tests/tests.js#L710, which will be 17 bytes.